tiatoolbox/wsicore/wsireader.py

```diff
@@ -2389,6 +2314,207 @@ def read_bounds(
         return im_region
 
 
+class DICOMWSIReader(WSIReader):
+    wsidicom = None
+
+    def __init__(
+        self,
+        input_img: Union[str, pathlib.Path, np.ndarray],
+        mpp: Optional[Tuple[Number, Number]] = None,
+        power: Optional[Number] = None,
+    ) -> None:
+        from wsidicom import WsiDicom
+
+        super().__init__(input_img, mpp, power)
+        self.wsi = WsiDicom.open(input_img)
+
+    def _info(self) -> WSIMeta:
+
+        level_dimensions = [
+            (level.size.width, level.size.height) for level in self.wsi.levels
+        ]
+        level_downsamples = [
+            np.mean(
+                [
+                    level_dimensions[0][0] / level.size.width,
+                    level_dimensions[0][1] / level.size.height,
+                ]
+            )
+            for level in self.wsi.levels
+        ]
+        dataset = self.wsi.datasets[0]
+        mm_per_pixel = dataset.pixel_spacing
+        mpp = (mm_per_pixel.width * 1e3, mm_per_pixel.height * 1e3)
```

Comment:
micron or mili ? micron should use `um`

Reply:
The metadata stores it in mm so here I am converting to um (mpp)